selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/RenameProjectTest.java

```diff
@@ -69,7 +78,15 @@ public void renameProjectTest() {
 
     // Wait that project renamed and folder has project type
     projectExplorer.waitItem(NEW_PROJECT_NAME);
-    projectExplorer.waitItemInvisibility(PROJECT_NAME);
+
+    // For ensuring of #12000 bug checking, folder with old name does nor appear immediately
+    WaitUtils.sleepQuietly(20);
+
+    try {
+      projectExplorer.waitItemInvisibility(PROJECT_NAME);
+    } catch (TimeoutException ex) {
+      fail("Known permanent issue: https://github.com/eclipse/che/issues/12000");
```

Comment:
We have to mark test as being _UNDER_REPAIR_, isn't it?

Reply:
Yes, definitely